src/modules/tests/pass-completion/pass-completion.cat-be.reducer.ts

```diff
@@ -1,12 +1,12 @@
 
 import * as passCompletionActions from './pass-completion.actions';
-import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { PassCompletion } from '@dvsa/mes-test-schema/categories/common';
 import { createFeatureSelector } from '@ngrx/store';
```

Comment:
If there is no difference between Cat B and B+E could we not move stuff into common and get rid of the category specific reducers and selectors

Reply:
We sure can